src/components/Web3Status/ConnectWalletPopover.tsx

```diff
@@ -206,10 +235,20 @@ export const Item = ({ id, onClick, name, icon, link, active }: ItemProps) => {
           {getContent()}
         </ListButton>
       ) : (
-        <ListButton disabled={active} onClick={onClick}>
+        <ListButton disabled={isActive} onClick={onClick}>
           {getContent()}
         </ListButton>
       )}
     </ListItem>
   )
 }
+
+export const Disconnect = ({onClick}: DisconnectProps) => {
+  return (
+    <ListFooter>
+      <ListButton onClick={onClick}>
+        DISCONNECT WALLET
+      </ListButton>
+    </ListFooter>
+  )
+}
```

Comment:
Also regarding this whole components I think its not neccesary to put it into function like this and also could it be done without the list footer and handle everything in listButton and have some generic wrapper similar to how walletAction was done...